packages/@mantine/charts/src/LineChart/LineChart.test.tsx

```diff
@@ -1,6 +1,19 @@
-import { patchConsoleWarn, tests } from '@mantine-tests/core';
+import React from 'react';
+import { patchConsoleWarn, render, tests } from '@mantine-tests/core';
 import { LineChart, LineChartProps, LineChartStylesNames } from './LineChart';
 
+jest.mock('recharts', () => {
```

Comment:
Why is this required? Other tests work correctly without it.

Reply:
Hi @rtivital! Thank you for the PR review. 

This mock enables LineChart to be rendered in the DOM. Without this mock we are not able to do assertions that elements are in the DOM and rendered correctly. Prior to this the unit tests did not assert that any elements were rendering correctly for LineChart. 

Before tests.itSupportsSystemProps were asserting that system properties were supported and this particular test did not rely on LineChart mounting to be able to test elements rendering correctly.